warehouse/query-core/src/main/java/datawave/query/predicate/EventDataQueryExpressionFilter.java

```diff
@@ -103,15 +101,26 @@ protected boolean apply(Key key, boolean update) {
             throw new RuntimeException("The EventDataQueryExpressionFilter was not initialized");
         }
 
-        final DatawaveKey datawaveKey = new DatawaveKey(key);
-        final String fieldName = JexlASTHelper.deconstructIdentifier(datawaveKey.getFieldName(), false);
+        final String fieldName = getFieldNameFromKey(key);
```

Comment:
Is this used in more than one location?  Otherwise, why refactor?

Reply:
The idea was to use a key parser and avoid the datawave key, but exploratory refactors in that direction did not prove useful.